### PR TITLE
114 teeth brushing

### PR DESCRIPTION
**Pull request by 102101141**
Made bubble particle system (PS, texture, reflections).
Added ToolVFX script.
Added ToolFVX script to brush prefab.
Added everything to InputManager.

### CODE REVIEW

File: MonstieWash/Assets/Scripts/VFX/ToolFX.cs [hunk 1]
```diff
@@ -0,0 +1,43 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+
+public class ToolFX : MonoBehaviour
+{
+    [SerializeField] private ParticleSystem particlesOnUse;
+    ParticleSystem myParticles;
```
Comment (CrispyKorn) [inline MonstieWash/Assets/Scripts/VFX/ToolFX.cs:9]: Just convention stuff.
```suggestion
    private ParticleSystem m_myParticles;
```

File: MonstieWash/Assets/Scripts/VFX/ToolFX.cs [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+
+    private void OnDisable()
+    {
+        InputManager.Inputs.OnActivate_Started -= StartParticles;
+        InputManager.Inputs.OnActivate_Ended -= StopParticles;
+    }
+
+        /// <summary>
+    /// Called every frame until the activate button is released.
+    /// </summary>
```
Comment (CrispyKorn) [inline MonstieWash/Assets/Scripts/VFX/ToolFX.cs:31]: Can remove these lines (they were from the eraser script).

```suggestion
```

File: MonstieWash/Assets/Scripts/VFX/ToolFX.cs [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+    private void StartParticles()
+    {
+        Debug.Log("Subscribed");
```
Comment (CrispyKorn) [inline MonstieWash/Assets/Scripts/VFX/ToolFX.cs:34]: Don't need this anymore.
```suggestion
```

**Review by CrispyKorn — CHANGES_REQUESTED** [review]
All looks good, just a couple tweaks and also don't forget to also change the simulation space on the Bubble_Sub_PS from local to world space so that when the bubbles pop those particles are also in world space. Other than that, all good!

**Comment by 102101141** [comment]
Had to do further modification of this code that rendered this PR obsolete: ToolFX script couldn't subscribe to the InputManager properly because it had no context of whether the tool was being held or not, and thus the particles would play even while the tool was unattended. New PR will fix this problem by including the functionality in the eraser script.

**Pull request closed without merging**